src/main/java/de/tum/in/www1/artemis/web/rest/ComplaintResource.java

```diff
@@ -257,7 +264,7 @@ public ComplaintResource(AuthorizationCheckService authCheckService, ExerciseRep
      * @param complaintType the type of complaints we are interested in
      * @return the ResponseEntity with status 200 (OK) and a list of complaints. The list can be empty
      */
-    @GetMapping("/complaints")
+    @GetMapping("/complaints") // TODO: should be "courses/{courseId}/complaints"
```

Comment:
we should indicate in the url that not all complaints but the ones for one specific tutor are retrieved